### PR TITLE
Fix default text colors within the Cover Block

### PR DESCRIPTION
**Pull request by kjellr**
Followup from https://github.com/WordPress/twentytwentyone/pull/404. 

That PR overrides some of the existing logic for changing the default text color. This fixed an issue with the default cover background, but it caused issues with any light-colored background colors. 

The core issue appears to just be that the theme was using the wrong variable for the text color (`--cover--color-background` instead of `--cover--color-foreground`). This PR removes the changes from #404, and makes that small update. 

---

Editor

Before|After
---|---
gutenberg test_wp-admin_post php_post=1 action=edit (1)|gutenberg test_wp-admin_post php_post=1 action=edit

Front End

Before|After
---|---
gutenberg test__p=1 (1)|gutenberg test__p=1

### CODE REVIEW

**Review by aristath — APPROVED** [review]
Fix confirmed :+1: 
I'll wait for a review by @JustinyAhin before merging since this PR basically reverts #404, so I may have missed something.

**Review by JustinyAhin — APPROVED** [review]
Looks good to me too.
Thanks :).

**Pull request merged**
